lib/drive-get-experiment.js

```diff
@@ -0,0 +1,52 @@
+const Hyperdrive = require('hyperdrive')
+const Corestore = require('corestore')
+const Experiment = require('./experiment')
+
+const main = async () => { 
+  await drive.put('/blob.txt', Buffer.from('example'))
+
+  const buffer = await drive.get('/blob.txt')
+  console.log(buffer)
+
+  const entry = await drive.entry('/blob.txt')
+  console.log(entry)
+}
+
+class DriveGetExperiment extends Experiment {
+  constructor (tmpDir, { nrBlocks, blockByteSize }) {
+    super(tmpDir, 'Drive Get Hyperdrive experiment')
+
+    this.store = new Corestore(this.tmpDir)
+    this.drive = new Hyperdrive(this.store)
+    
+    this.nrBlocks = nrBlocks
+    this.blockByteSize = blockByteSize
+
+    this.block = 'a'.repeat(this.blockByteSize)
+  }
+
+  async _setup () {
+    if (this.closing) return
+    for (let i = 0; i < this.nrBlocks; i+=1) {
+      if(this.closing) return
+      await this.drive.put(`/blob${i}.txt`, Buffer.from(this.block))
+    }
+    await this.drive.ready()
+ }
+
+  async _runExperiment () {
+    const demoProms = []
+    for (let i = 0; i < this.nrBlocks; i+=1) {
+      if(this.closing) return
+      await this.drive.get(`/blob${i}.txt`)
+    }
```

Comment:
Good to add a sanity check here that the result of the `drive.get()' is not nullish
```
const content = await this.drive.get(...)
if (content == null) throw new Error('Bug in this code: nullish entry in the hyperdrive')
```

Reply:
`npm test` seems to complain about stderr output when i add this one.

Reply:
added the sanity check in a similar fashion.